strapi/.env.example

```diff
@@ -0,0 +1,6 @@
+HOST=0.0.0.0
```

Comment:
I always wondered, is there difference between using this host and 127.0.0.1 ?

Reply:
In theory, 127.0.0.1 is a loopback-only address, and 0.0.0.0 should be non-routable. In practice, it depends on the server implementation. Some will reject external requests if bound to 127.0.0.1 while others will work fine. However 0.0.0.0 is somewhat of standard to specify "listen and accept on all network interfaces", even when requests are external, e.g. definitely for public-facing servers.